spec/system/grant_permissions/grant_permissions_spec.rb

```diff
@@ -66,6 +66,16 @@
         expect(page).to have_content 'There must be at least one admin on the grant'
         expect(@grant_admin_role.role).to eql('admin')
       end
+
+      scenario 'can change notification level for submissions' do
+        visit edit_grant_grant_permission_path(@grant, @grant_admin_role)
+        expect(@grant_admin_role.submission_notification).to eql false
+        find(:css, "#grant_permission_submission_notification").set(true)
+        click_button 'Update'
+        visit edit_grant_grant_permission_path(@grant, @grant_admin_role)
+        @grant_admin_role.reload
+        expect(@grant_admin_role.submission_notification).to eql true
+      end
```

Comment:
Since we know there are other possible notification settings (e.g. review completion), consider nesting this spec in a `mailers` context